### DISCUSSION TITLE
Issue when grouping with multiple columns

### DISCUSSION THREAD

**Discussion by chanm003** [KevinVandy/mantine-react-table#294] [Q&A] [answered]
### mantine-react-table version

2.0.0-beta.0

### react & react-dom versions

18.2.0

### Describe the bug and the steps to reproduce it

I believe there is a bug, and it exists in the documentation as well: 
[Grouping example](https://v2.mantine-react-table.com/docs/examples/aggregation-and-grouping)

### Minimal, Reproducible Example - (Optional, but Recommended)

When grouping by State, Arizona reports 8:
![when-grouping-by-state-arizona-shows-eight](https://github.com/KevinVandy/mantine-react-table/assets/2539430/87a5f48a-55e7-49c2-8c8d-e982686fee5c)

When grouping by State then Gender, the sub grouping accurately reports 6 Arizona males, 2 Arizona females.   However, the total for Arizona went from reporting 8 to now 2
![when-grouping-by-state-then-gender-expect-8-for-arizona](https://github.com/KevinVandy/mantine-react-table/assets/2539430/e3a30a13-1c73-443d-930f-da6bbefc352d)

### Do you intend to try to help solve this bug with your own PR?

None

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by chanm003** [comment]
Okay, I think I now see where the disconnect is.   Arizona reports "2" because there are two unique genders in the Arizona group, not eight unique genders.

Is this the "universal standard" for reporting multi-level grouping?

**Reply by alessandrojcm** [reply, marked as answer]
> Okay, I think I now see where the disconnect is. Arizona reports "2" because there are two unique genders in the Arizona group, not eight unique genders.
> 
> Is this the "universal standard" for reporting multi-level grouping?

Hey, yes that is correct. Grouping happens at the first sub-row level, so you see (2) because there are 2 unique genders and then under Male you see 6 and under Female you see 2. You can always [define your own aggregation function](https://tanstack.com/table/v8/docs/api/features/grouping#table-options) if you need something different.
